cms/earn/strategies.json

```diff
@@ -2474,6 +2474,51 @@
         }
       ],
       "categories": ["Correlated"]
+    },
+    {
+      "id": "osmo1qngwlgzt0r3fdg6zgln5wa6mr5c7t6fek3qqgxesujjak5ds747sy6qu2f",
+      "name": "USDC.axl/USDC Locust",
+      "platform": "Margined",
+      "type": "LP (Vault)",
+      "method": "margined-locust-grid",
+      "link": "https://trade.margined.io/vaults/grid-usdcaxl-usdc",
+      "tvl": "https://vaults.margined.io/api/v1/osmo1qngwlgzt0r3fdg6zgln5wa6mr5c7t6fek3qqgxesujjak5ds747sy6qu2f/tvl",
+      "apr": "https://vaults.margined.io/api/v1/osmo1qngwlgzt0r3fdg6zgln5wa6mr5c7t6fek3qqgxesujjak5ds747sy6qu2f/apr",
+      "geoblock": "https://vaults.margined.io/api/v1/osmo1qngwlgzt0r3fdg6zgln5wa6mr5c7t6fek3qqgxesujjak5ds747sy6qu2f/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=631885567#gid=631885567",
```

Comment:
Please use this URL for the public-facing sheet: https://docs.google.com/spreadsheets/d/1xVvxWgUAagQbtKTZgQPXSpSMbpW2RyKxEtfIbzowtKM/edit?gid=2069375532#gid=2069375532

Reply:
@JeremyParish69 this is updated. thanks!